flow/util/genMetrics.py

```diff
@@ -256,6 +256,15 @@ def extract_metrics(
     # =========================================================================
     merge_jsons(logPath, metrics_dict, "2_*.json")
 
+    extractTagFromFile(
```

Comment:
You should not need to scrape the OR log file, just have rsz report the metric at the same point this message is being generated.  Use Logger::metric. eg
```
  logger_->metric("design__instance__count__swapped_arithmetic_operator",
                  swapped_count);
```